src/collections/members/ashish-tiwari/index.mdx

```diff
@@ -0,0 +1,17 @@
+---
+name: Ashish Tiwari
+position: Contributor, Community
```

Comment:
I think we can keep this to only `Contrbutor` for now.

Reply:
```suggestion
position: Contributor
```

Reply:
Noted.